src/Symfony/Component/Mime/Encoder/Base64Encoder.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mime\Encoder;
+
+/**
+ * @final
```

Comment:
conflicts with the fact that we extend from it - DebugClassLoader will report it :)
since there is only one method and it's part of the interface, we can remove it, isn't it?

Reply:
Removed